index.html

```diff
@@ -75,8 +75,10 @@ <h2>Heading</h2>
           <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
        </div>
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Vestibulum id ligula porta felis euismod semper. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.</p>
+          <h2>Content Presentation Guidelines</h2>
+          <p><li>1) Registered Users have the ability to upload both images as well as a file that has text,</li>
+             <li>2) Registered users are provided the acceptable file formats for uploading either images or files with text,</li>
+             <li>3) The user interface components for a registered user must be operable and allow the user to discuss any poem with other users.</li></p>
```

Comment:
Here you'll want to change the closing paragraph tag to a closing ordered list tag:
```suggestion
             <li>3) The user interface components for a registered user must be operable and allow the user to discuss any poem with other users.</li>
             </ol>
```